mod/workspace/cache.js

```diff
@@ -1,36 +1,36 @@
-const file = require('../provider/file')
+const file = require("../provider/file");
 
-const cloudfront = require('../provider/cloudfront');
+const cloudfront = require("../provider/cloudfront");
 
-const http = require('./httpsAgent')
+const http = require("./httpsAgent");
 
 const getFrom = {
-  'https': ref => http(ref),
-  'file': ref => file(ref.split(':')[1]),
-  'cloudfront': ref => cloudfront(ref.split(':')[1]),
-}
+  https: (ref) => http(ref),
+  file: (ref) => file(ref.split(":")[1]),
+  cloudfront: (ref) => cloudfront(ref.split(":")[1]),
+};
 
-const assignTemplates = require('./assignTemplates')
+const assignTemplates = require("./assignTemplates");
 
-const defaults = require('./defaults');
+const defaults = require("./defaults");
 
-const assignDefaults = require('./assignDefaults');
+const assignDefaults = require("./assignDefaults");
 
 let workspace = null;
 
-const { nanoid } = require('nanoid');
+const { nanoid } = require("nanoid");
 
-const logger = require('../logger');
+const logger = require("../logger");
```

Comment:
Not mandatory I guess but can we put all require statements at the top of the file?